pkg/cache/cache_test.go

```diff
@@ -143,46 +144,46 @@ func TestCacheWriteCacheDirExists(t *testing.T) {
 }
 
 func TestStoreRelocationMapping(t *testing.T) {
-	tests := []struct {
+	home, err := os.UserHomeDir()
+	assert.NoError(t, err)
+	tests := map[string]struct {
 		name              string
 		relocationMapping relocation.ImageRelocationMap
 		tag               cnab.OCIReference
 		bundle            cnab.ExtendedBundle
 		wantedReloPath    string
 		err               error
 	}{
-		{
-			name:   "relocation file gets a path",
+		"relocation file gets a path": {
 			bundle: cnab.ExtendedBundle{},
 			tag:    kahn1dot01,
 			relocationMapping: relocation.ImageRelocationMap{
 				"asd": "asdf",
 			},
-			wantedReloPath: path.Join("/.porter/cache", kahn1dot0Hash, "cnab", "relocation-mapping.json"),
+			wantedReloPath: path.Join(home+"/.porter/cache", kahn1dot0Hash, "cnab", "relocation-mapping.json"),
 		},
-		{
-			name:           "no relocation file gets no path",
+		"no relocation file gets no path": {
 			tag:            kahnlatest,
 			bundle:         cnab.ExtendedBundle{},
 			wantedReloPath: "",
 		},
 	}
 
-	for _, tc := range tests {
-		t.Run(tc.name, func(t *testing.T) {
+	for name, test := range tests {
+		test := test
```

Comment:
```suggestion
	for name, tc := range tests {
		tc := tc
```
Minor nit so this is similar with other tests

Reply:
This has been updated on the latest push.